commands/core/install.go

```diff
@@ -54,9 +54,9 @@ func PlatformInstall(ctx context.Context, req *rpc.PlatformInstallRequest,
 		return nil, err
 	}
 
-	_, err = commands.Rescan(req.GetInstance().GetId())
-	if err != nil {
-		return nil, err
+	_, status := commands.Init(&rpc.InitRequest{Instance: &rpc.Instance{Id: req.Instance.Id}})
```

Comment:
This may be changed to:
```suggestion
	_, status := commands.Init(&rpc.InitRequest{Instance: req.Instance})
```